app/coffee/Notifications.coffee

```diff
@@ -22,25 +22,36 @@ module.exports =
 				logger.log number:number, user_id:user_id, key:notification.key, "alredy has notification key for user"
 				callback number
 			else
-				doc = 
+				doc =
 					user_id: ObjectId(user_id)
 					key: notification.key
 					messageOpts: notification.messageOpts
 					templateKey: notification.templateKey
+					expires: notification.expires || false
+				# ttl index on `expiresFrom` field
+				if doc.expires
+					doc.expiresFrom = new Date()
```

Comment:
doc.expires would be a boolean, indicating that the doc should expire, while expiresFrom would be the actual ttl'd date column. TTL indexes in Mongo only work on Date fields, will add a comment to that effect